crates/bevy_sprite/src/render/mod.rs

```diff
@@ -590,6 +597,38 @@ pub fn queue_sprites(
                 });
             }
         }
+
+        for (batch_entity, sprite_entities) in extracted_batches.batches.iter() {
+            if !view_entities.contains(batch_entity.index() as usize) {
```

Comment:
This could use a bit of a comment: it's really quite unusual to care about the index of an Entity.

Reply:
The original code wasn't commented either, I'm confident these changes work and don't mess anything up but I don't know enough about the rendering internals to be explaining anything to anyone I think.